templates/pages/generic_in_modal.html.twig

```diff
@@ -39,9 +39,6 @@
     ]
 ) }}
 
-{{ object.showForm(
-    id,
-    form_options
-) }}
+{% set _ = object.showForm(id, form_options) %}
```

Comment:
This seems odd to me

Reply:
object.showForm() returns a value, a boolean.
Just using the call make a '1' appear in the browser, because of the returned value.
This tricks solve this problem.

Reply:
Wouldn't `{% object.showForm(id, form_options) %}` do the same ?

Reply:
because of the return value of showForm() which is displayed.

Reply:
`{%` execute a function and won't display the return value. Maybe you tested with `{{` before, which does output the return ?

Reply:
Use `{% do ... %}` to block outputting the return value.
https://twig.symfony.com/doc/3.x/tags/do.html